packages/colony-starter-basic/src/metamask.js

```diff
@@ -1,18 +1,24 @@
-// Import example
+// Import prerequisites
+const chalk = require('chalk');
+
+// Import examples
 const connectMetamask = require('./examples/connectMetamask');
 
-// Define connectMetamaskExample
-const connectMetamaskExample = async () => {
+// Logging to help keep track of where we are in the script
+const log = (msg) => console.log(chalk.cyan('\n' + msg + '\n'));
```

Comment:
Maybe it would be better to import the `log` function from a single location, and then replace all instances of `console.log` with it? A bit of find and replace fun, but it will be easier to maintain, I promise :)

Reply:
Can do!